engines/bastion_katello/app/assets/javascripts/bastion_katello/hosts/host.factory.js

```diff
@@ -30,6 +30,10 @@ angular.module('Bastion.hosts').factory('Host',
             return _.isString(this.operatingsystem_name) && (this.operatingsystem_name.indexOf("Debian") >= 0 || this.operatingsystem_name.indexOf("Ubuntu") >= 0);
         };
 
+        resource.prototype.rhelLifecycleExpireSoon = function() {
```

Comment:
Do you like this format better or the React way of doing things?

Reply:
React is easier to understand to me.